vm/image.cpp

```diff
@@ -253,7 +252,7 @@ void factor_vm::load_image(vm_parameters* p) {
   load_data_heap(file, &h, p);
   load_code_heap(file, &h, p);
 
-  safe_fclose(file);
```

Comment:
``safe_fclose`` is wrong anyway -- you're not allowed to retry on ``EINTR``
on ``flclose`` or ``close``.

On Wed, Jun 3, 2015 at 11:24 AM, John Benediktsson <notifications@github.com
> wrote:

> In vm/image.cpp
> <https://github.com/slavapestov/factor/pull/1334#discussion_r31653622>:
>
> > @@ -253,7 +252,7 @@ void factor_vm::load_image(vm_parameters* p) {
> >    load_data_heap(file, &h, p);
> >    load_code_heap(file, &h, p);
> >
> > -  safe_fclose(file);
>
> Why change this one? To hide io-error if close fails?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/slavapestov/factor/pull/1334/files#r31653622>.
>